### DISCUSSION TITLE
How to search for taxonomy entries within a page

### DISCUSSION THREAD

**Discussion by fernandogiroto** [statamic/cms#8542] [Q&A]
Hello! 
I'm trying to get the entrys of the taxonomies within a page but I'm not able to get them. I saw that I have this method but it does not bring me the entrys and it is not a collection either.

![Captura de Tela 2023-08-03 às 11 12 34](https://github.com/statamic/cms/assets/23166586/dd51968b-ff33-4854-ad7e-66225f110aff)

**Comment by edalzell** [comment]
What do you mean "search for entries within a page"? Can you give an example of what you actually want to happen?
